homu/server.py

```diff
@@ -29,8 +29,16 @@ def index():
 
 @get('/queue/<repo_label>')
 def queue(repo_label):
-    repo = g.repos[repo_label]
-    pull_states = sorted(g.states[repo_label].values())
+    if repo_label == 'all':
+        labels = list(g.repos.keys())
```

Comment:
Just curious, why did you use `list()` here? To match the type with `repo_label.split('+')` below?